### PR TITLE
Make hpi:bundle-plugins fail if you forgot the <type> tag

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

**Comment by tfennelly** [comment]
:+1: 

Assuming it works ... don't see a test.

File: src/it/bad-bundle/invoker.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+invoker.goals=clean generate-resources
+invoker.buildResult = failure
```
Comment (stephenc) [inline src/it/bad-bundle/invoker.properties]: @tfennelly here is the test

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by jglick** [comment]
Oh, thank you, this will save a lot of headaches.

**Pull request merged**